internal/identity/identitymanager.go

```diff
@@ -44,14 +44,15 @@ type Manager interface {
 	ResolveInputSigningKey(ctx context.Context, inputKey string, keyNormalizationMode int) (signingKey string, err error)
 	ResolveQuerySigningKey(ctx context.Context, inputKey string, keyNormalizationMode int) (signingKey string, err error)
 	ResolveIdentitySigner(ctx context.Context, identity *core.Identity) (parentSigner *core.SignerRef, err error)
+	ResolveMultipartyRootVerifier(ctx context.Context) (*core.VerifierRef, error)
+	ResolveMultipartyRootOrg(ctx context.Context) (*core.Identity, error)
```

Comment:
I'm questioning why we need this.
The only time we should need to do a `resolve` I believe, is when we need the signing key. But this call only returns the `core.Identity`

Reply:
Yea, I looked at all the remaining paths that use this call, and they either 1) require the DID only (without key resolution) or 2) are actually performing key resolution twice, because this method performs the resolution and another path performs it again.

I'll remove this method altogether.